storage/postgres/migrations/20250401065837_add_reviews.sql

```diff
@@ -0,0 +1,18 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE reviews (
+    id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    address TEXT NOT NULL,
+    rating INT CHECK (rating BETWEEN 1 AND 5),
+    comment TEXT NOT NULL,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    plugin_id UUID NOT NULL REFERENCES plugins(id),
```

Comment:
`plugin_id UUID NOT NULL` 
`REFERENCES plugins(id)` this is already specified in the fk